lib/std/crypto/keccak_p.zig

```diff
@@ -294,12 +294,17 @@ pub fn State(comptime f: u11, comptime capacity: u11, comptime rounds: u5) type
 
         /// Permute the state
         pub fn permute(self: *Self) void {
+            if (mode == .Debug) {
```

Comment:
Other parts of the standard library generally enable safety checks like these in `ReleaseSafe` as well, FWIW.